web/src/app/webview/src/contextManager.ts

```diff
@@ -71,15 +71,26 @@ export default class ContextManager extends ContextManagerBase<WebviewConfigurat
     return null;
   }
 
+  protected getFallbackCodes(): KeyboardStub {
+    // Fallback behavior - we're embedded in a touch-device's webview, so we need to keep a keyboard visible.
+    return this.keyboardCache.defaultStub;
+  }
```

Comment:
Can we call this `getFallbackStub` instead?

```suggestion
  protected getFallbackStub(): KeyboardStub {
    // Fallback behavior - we're embedded in a touch-device's webview, so we need to keep a keyboard visible.
    return this.keyboardCache.defaultStub;
  }
```